tests/integration/test_integration.py

```diff
@@ -745,3 +746,28 @@ def test_inin_024_port_env_success(dash_duo):
     dash_duo.start_server(app, port="12345")
     assert dash_duo.server_url == "http://localhost:12345"
     dash_duo.wait_for_text_to_equal("#out", "hi")
+
+
+def nested_app(server, path, text):
+    app = Dash(__name__, server=server, url_base_pathname=path)
+    app.layout = html.Div(id="out")
+
+    @app.callback(Output("out", "children"), [Input("out", "n_clicks")])
+    def out(n):
+        return text
+
+    return app
+
+
+def test_inin025_url_base_pathname(dash_br, dash_thread_server):
+    server = flask.Flask(__name__)
+    app = nested_app(server, "/app1/", "The first")
+    nested_app(server, "/app2/", "The second")
```

Comment:
Shockingly we had no tests for multiple apps in one server... thanks for pointing this out @christianwengert!

`dash_duo` currently needs an app running at `/` or it will time out; a clean fix for this wasn't obvious to me, so I just used `dash_br` and `dash_thread_server` (effectively the two pieces that make the "duo" of `dash_duo`) separately.